JASP-Engine/JASP/R/selectionmodels.R

```diff
@@ -77,22 +81,36 @@ SelectionModels <- function(jaspResults, dataset, options, state = NULL) {
   }
   
   .hasErrors(dataset               = dataset,
-             type                  = c("infinity", "observations", "variance"),
+             type                  = c("infinity", "observations"),
              observations.amount   = "< 2",
              exitAnalysisIfErrors  = TRUE)
   
-  .hasErrors(dataset               = dataset,
-             type                  = c("negativeValues"),
-             negativeValues.target = options[["input_SE"]],
-             exitAnalysisIfErrors  = TRUE)
-  
+  if(options[["input_SE"]] != ""){
```

Comment:
```suggestion
  if (options[["input_SE"]] != "") {
```

Reply:
And further where appropriate (see the style guide: https://github.com/jasp-stats/jasp-desktop/blob/stable/Docs/development/r-style-guide.md)

Reply:
done